evm_loader/program-macro/src/lib.rs

```diff
@@ -29,7 +32,7 @@ pub fn operators_whitelist(tokens: TokenStream) -> TokenStream {
         .list
         .iter()
         .map(LitStr::value)
-        .map(|key| bs58::decode(key).into_vec().unwrap())
+        .map(|key| bs58::decode(key).into_vec().expect("Pubkey is base64 encoded"))
```

Comment:
Same as above. Better to write something like "Expected pubkey to be base64 encoded string".

Reply:
This code is executed at compile time and the message is more than informative for a developer trying to build a project.